extensions/wikia/CommunityPage/templates/insightsModule.mustache

```diff
@@ -1,11 +1,8 @@
 <section class="community-page-insights-module" data-tracking="community-page-insights-{{type}}">
-	<h3 class="community-page-insights-module-header control">{{title}}</h3>
-	<h3 class="community-page-insights-module-header medium">{{mediumtitle}}</h3>
-	<h3 class="community-page-insights-module-header high lite">{{hightitle}}</h3>
-	<p class="community-page-insights-module-description control">{{description}}</p>
-	<p class="community-page-insights-module-description medium">{{mediumdescription}}</p>
-	<p class="community-page-insights-module-description high">{{highdescription}}</p>
-	<p class="community-page-insights-module-description lite"></p>
+	<h3 class="community-page-insights-module-header">{{title}}</h3>
+    {{#description}}
+        <p class="community-page-insights-module-description">{{description}}</p>
```

Comment:
`community-page-insights-module-description` class has `clear:both` property, so if we are not displaying it everywhere the layout is a little bit broken now.